src/components/welcome-page/Welcome.scss

```diff
@@ -0,0 +1,90 @@
+@import "../../styles/variables.scss";
+
+.main-content {
+    position: relative;
+    display: flex;
```

Comment:
add Mixins.scss and place inside it 

centered-flex with

display: flex;
 justify-content: center;
    align-items: center;